cves/kernel/CVE-2012-6545.yml

```diff
@@ -292,8 +296,9 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    This mistake was extremely low level. It dealt with memory management.
```

Comment:
Consider revising the note here.  Low-level processes do perform IPC.

Reply:
I will specify it something other than "low-level".